core/common/src/main/java/alluxio/AbstractClient.java

```diff
@@ -160,6 +195,10 @@ protected void afterConnect() throws IOException {
     // Empty implementation.
   }
 
+  protected Set<GrpcNetworkGroup> getAdditionalNetworkGroups() {
+    return ImmutableSet.of();
+  }
```

Comment:
This provides a control handle for inheriting classes to designate additional channels they need.